libtuf-server/src/main/scala/com/advancedtelematic/libtuf_server/data/Messages.scala

```diff
@@ -29,4 +30,28 @@ object Messages {
   implicit val packageStorageUsageDecoder: Decoder[PackageStorageUsage] = deriveDecoder
 
   implicit val packageStorageUsageMessageLike = MessageLike[PackageStorageUsage](_.namespace)
+
+  case class DeviceUpdateReport(namespace: Namespace, device: DeviceId, updateId: UpdateId, timestampVersion: Int,
+                                operationResult: Map[EcuSerial, OperationResult], resultCode: Int)
+
+  implicit val operationResultEncoder: Encoder[OperationResult] = deriveEncoder
+  implicit val operationResultDecoder: Decoder[OperationResult] = deriveDecoder
+
+  implicit val deviceUpdateReportEncoder: Encoder[DeviceUpdateReport] = deriveEncoder
+  // For backwards compatibility reasons we have a decoder that can parse DeviceUpdateReport
```

Comment:
when do we need this? Is this still the case?

Reply:
probably not, simplified